src/Hosting/Hosting/src/Internal/StartupLoader.cs

```diff
@@ -228,15 +228,17 @@ public static Type FindStartupType(string startupAssemblyName, string environmen
             if (string.IsNullOrEmpty(startupAssemblyName))
             {
                 throw new ArgumentException(
-                    string.Format("A startup method, startup type or startup assembly is required. If specifying an assembly, '{0}' cannot be null or empty.",
-                    nameof(startupAssemblyName)),
-                    nameof(startupAssemblyName));
+                    string.Format(
+                        CultureInfo.CurrentCulture,
```

Comment:
Why current culture here instead of invariant culture?

Reply:
Exception messages and are meant to be formatted using your current culture. The rule of thumb is - when formatting strings that is user visible use `CurrentCulture`, when formatting or parsing for machines, use `InvariantCulture`